### PR TITLE
HTML-739: Expose FormSubmissionActions#transitionToState(..) as a public util method.

### PR DESCRIPTION
**Pull request by samuelmale**
Ticket -> https://issues.openmrs.org/browse/HTML-739

### CODE REVIEW

**Comment by samuelmale** [comment]
I see there is zero test coverage(not even a test class) for members of the `FormSubmissionActions` class. I think we need a ticket for this?

File: api-2.3/src/test/java/org/openmrs/htmlformentry/element/ConditionTagTest.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public void testResults(SubmissionResults results) {
 				
 				results.assertNoErrors();
 				Assert.assertEquals(2, conditions.length);
-								
+
```
Comment (mks-d) [inline api-2.3/src/test/java/org/openmrs/htmlformentry/element/ConditionTagTest.java:82]: Maybe untouch this file for the clarity of the PR?

Comment (samuelmale) [reply api-2.3/src/test/java/org/openmrs/htmlformentry/element/ConditionTagTest.java:82]: Yeah sure, that happened when I ran a -> `mvn clean package`

Comment (samuelmale) [reply api-2.3/src/test/java/org/openmrs/htmlformentry/element/ConditionTagTest.java]: @mks-d Resolved.

**Pull request merged**